Cargo.toml

```diff
@@ -181,20 +181,24 @@ walkdir = "2"
 xsalsa20poly1305 = "0.9"
 zstd-safe = "6.0.5"
 
+# hamtv0 specific deps
+forest_hash_utils = "0.1"
+serde_bytes = { package = "cs_serde_bytes", version = "0.12" }
```

Comment:
This can stay if there's a good reason to keep it. Write a comment explaining why it cannot be replaced with `fvm_ipld_encoding`.

Reply:
This is not needed for this port, removed.